src/litserve/specs/openai.py

```diff
@@ -70,22 +70,17 @@ class ChatCompletionResponse(BaseModel):
     usage: UsageInfo
 
 
-class Delta(BaseModel):
-    role: str
-    content: str
-
-
 class StreamingChoice(BaseModel):
     index: int
-    delta: Delta
-    logprobs: Optional[dict]
+    delta: ChatMessage
+    logprobs: Optional[dict] = None
     finish_reason: Optional[Literal["stop", "length"]]
 
 
 class ChatCompletionChunk(BaseModel):
-    id: str
-    object: str
-    created: int
+    id: str = Field(default_factory=lambda: f"chatcmpl-{shortuuid()}")
+    object: str = "chat.completion"
```

Comment:
isn't this `"chat.completion.chunk"`?
see https://platform.openai.com/docs/api-reference/chat/create